src/test/java/core/basesyntax/OperationHandlerTest.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.operations.ExpenseOperationHandler;
+import core.basesyntax.service.operations.IncomeOperationHandler;
+import core.basesyntax.service.operations.OperationHandler;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class OperationHandlerTest {
+    private static final OperationHandler INCOME = new IncomeOperationHandler();
+    private static final OperationHandler EXPENSE = new ExpenseOperationHandler();
+    private static final String WATERMELON = "watermelon";
+    private static final String MELON = "melon";
+
+    @Test
+    public void operate_putToStorage_validData_ok() {
```

Comment:
```suggestion
    public void income_validData_ok() {
```
"income" is not the name of the test method, but in your case where you are testing two classes with the same methods, it would be more clear to rename your test methods in such a way where the first part is the operation type (class name)